<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salesreport</class>
 <widget class="QWidget" name="Salesreport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="salestable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>761</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2020</year>
     <month>3</month>
     <day>18</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2020</year>
     <month>3</month>
     <day>12</day>
    </date>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="memberType">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Executive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standard</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="filterType">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter by type</string>
   </property>
  </widget>
  <widget class="QLabel" name="revenue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Daily Revenue</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemsBought">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Items Purchased</string>
   </property>
  </widget>
  <widget class="QTableView" name="memberList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dailyShopper">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Today's Shoppers</string>
   </property>
  </widget>
  <widget class="QLabel" name="numStan">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Standard</string>
   </property>
  </widget>
  <widget class="QLabel" name="numExec">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Executive</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
